lib/livebook/config.ex

```diff
@@ -493,4 +501,31 @@ defmodule Livebook.Config do
     IO.puts("\nERROR!!! [Livebook] " <> message)
     System.halt(1)
   end
+
+  @doc """
+  Parses zero trust identity provider from env.
+  """
+  def identity_provider!(env) do
+    case System.get_env(env) do
+      "googleiap:" <> rest ->
+        {Livebook.ZTA.GoogleIAP, rest}
+
+      "cloudflare:" <> rest ->
+        {Livebook.ZTA.Cloudflare, rest}
+
+      nil ->
+        nil
+
+      _ ->
+        abort!("invalid configuration for identity provider")
+    end
+  end
+
+  @doc """
+  Returns if the identity data is readonly.
+  """
+  @spec identity_readonly?() :: boolean()
```

Comment:
Nitpick: this is the parsing section, let's move this function under `identity_provider` :)